README.md

```diff
@@ -108,6 +108,14 @@ By default, Altis Analytics will grant any role who can edit pages the ability t
 
 Define as false to disable the capability fallback to page capabilities.
 
+**`ALTIS_ANALYTICS_PINPOINT_BUCKET_ARN`**
+
+If you have the Pinpoint Kinesis Firehose set to back up data to S3 then defining this constant allows for data to be cleaned from the backups periodically too.
```

Comment:
This should have some commas. 

```suggestion
If you have the Pinpoint Kinesis Firehose set to back up data to S3, defining this constant allows for data to be cleaned from the backups periodically, too.
```

Reply:
Hm, the comma just before the last word in the sentence is weird to me. Saying the sentence out loud doesn't create a natural pause there. I think we can just remove the word "too".